CMakeLists.txt

```diff
@@ -10,9 +12,10 @@ add_library(${PROJECT_NAME}
         RTPacket.cpp
         RTProtocol.cpp
 )
+
 target_include_directories(${PROJECT_NAME} PUBLIC
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
-        $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/${PROJECT_NAME}>
+        $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
```

Comment:
Verify that this is correct.

Reply:
I made a local copy of master and merged this branch into it. It builds in the Ubuntu terminal on Windows, the CMake project generates and compiles properly, and when opening the solution all three projects also build.

With that said, I'm not sure if that completely satisfies as verification that it is correct.

Reply:
Testing yesterday with/without this change didn't _seem_ to make any difference. That said, I don't see this change as being correct or relevant to this PR, so I removed it.